example/lib/main.dart

```diff
@@ -31,6 +32,16 @@ class MyApp extends StatelessWidget {
                 decoration: InputDecoration(hintText: 'List type'),
               ),
 
+              /// sample1
```

Comment:
"sample 1” is duplicated. Please use a different name.
I suggest moving it to the bottom as “sample5.” 🙏